src/winetricks

```diff
@@ -1215,24 +1272,25 @@ w_download_to()
 
             # On Cygwin, .exe's must be marked +x
             case "$_W_file" in
-                *.exe) chmod +x "$_W_file" ;;
+                *.exe) chmod +x "$_W_file"
```

Comment:
Did you even test this?

I'm not reading further, this is just a waste of my time to review.

Reply:
Currently testing, Note that this is **DRAFT**